package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teamleader/ui",
   "description": "Teamleader UI library",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

Comment:
You shouldn't change the version in your PR. The version is usually changed in a separate release PR, because some other PR might get merged too that needs a different version increase.

Reply:
Alright, makes sense :)